Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -208,6 +206,21 @@ void GameFile::SaveCache()
   // TODO
 }
 
+QString GameFile::GetFileName() const
+{
+  return QFileInfo(m_path).fileName();
```

Comment:
The file was already using QFileInfo before this change. If you want I can reorder the commits?